naming/src/main/java/com/alibaba/nacos/naming/core/Cluster.java

```diff
@@ -300,21 +300,21 @@ public void updateIPs(List<Instance> ips, boolean ephemeral) {
         return new ArrayList<>(instanceMap.values());
     }
 
-    public List<Instance> subtract(Collection<Instance> a, Collection<Instance> b) {
-        Map<String, Instance> mapa = new HashMap<>(b.size());
-        for (Instance o : b) {
-            mapa.put(o.getIp() + ":" + o.getPort(), o);
+    public List<Instance> subtract(Collection<Instance> oldIp, Collection<Instance> ips) {
```

Comment:
In this line, I had changed it to a private function. So there is a conflict.
Please change it.

Reply:
@KomachiSion Hello, it has been solved. Please check it